js/config.js

```diff
@@ -0,0 +1,134 @@
+
+const mainContainer = creatElement('div',
+{ className: 'main-container' });
+
+const formContainer = creatElement('div',
+{ className: 'form-container' });
+
+const resultsContainer = creatElement('div',
+{ className: 'results-container', });
+
+
+window.onload = async function () {
+
+const marContainer = document.querySelector('.contain-marquee');
+const marquee = new Marquee(marContainer);
+
+marquee.start();
+
+const resultsSearch = new ResultsSearch(resultsContainer);
+
+const search = new Search(formContainer, resultsSearch);
+
+createMain();
+};
+
+
+function createMain() {
+mainContainer.append(formContainer, resultsContainer);
+document.body.append(mainContainer);
+}
+
+
+
+const getPercenInfo = profileNum => {
+  
+  let percentage = {};
+  percentage.num = profileNum;
+  percentage.text = parseFloat(percentage.num).toFixed(2) + '%';
+  percentage.Sign; percentage.Css;
+
+  if (percentage.num >= 0) {
+    percentage.Sign = '+'; percentage.Css = 'up0';
+  } else {
+    percentage.Sign = ''; percentage.Css = 'down0';
+  }
+  return percentage;
+};
+
+async function getJson(apiUrl) {
```

Comment:
good implemenation of you own "Fetch" function, but i would better name then getJson, something more genereic, like retriveDataFromServer